main.tf

```diff
@@ -15,50 +15,58 @@
  */
 
 locals {
-  account_billing = "${var.grant_billing_role && var.billing_account_id != ""}"
-  org_billing     = "${var.grant_billing_role && var.billing_account_id == "" && var.org_id != ""}"
-  prefix          = "${var.prefix != "" ? "${var.prefix}-" : ""}"
-  xpn             = "${var.grant_xpn_roles && var.org_id != ""}"
+  account_billing = var.grant_billing_role && var.billing_account_id != ""
+  org_billing     = var.grant_billing_role && var.billing_account_id == "" && var.org_id != ""
+  prefix          = var.prefix != "" ? "${var.prefix}-" : ""
+  xpn             = var.grant_xpn_roles && var.org_id != ""
 }
 
 # create service accounts
 resource "google_service_account" "service_accounts" {
-  count        = "${length(var.names)}"
+  count        = length(var.names)
   account_id   = "${local.prefix}${lower(element(var.names, count.index))}"
   display_name = "Terraform-managed service account"
-  project      = "${var.project_id}"
+  project      = var.project_id
 }
 
 # common roles
 resource "google_project_iam_member" "project-roles" {
-  count = "${length(var.project_roles) * length(var.names)}"
+  count = length(var.project_roles) * length(var.names)
 
-  project = "${element(
-    split("=>", element(var.project_roles, count.index % length(var.project_roles))
-  ), 0)}"
+  project = element(
+    split(
+      "=>",
+      element(var.project_roles, count.index % length(var.project_roles)),
+    ),
+    0,
+  )
 
-  role = "${element(
-    split("=>", element(var.project_roles, count.index % length(var.project_roles))
-  ), 1)}"
+  role = element(
+    split(
+      "=>",
+      element(var.project_roles, count.index % length(var.project_roles)),
+    ),
+    1,
+  )
 
   member = "serviceAccount:${element(
     google_service_account.service_accounts.*.email,
-    count.index / length(var.project_roles)
+    floor(count.index / length(var.project_roles)),
```

Comment:
What was the motivation for this change?

Reply:
@aaron-lane Without it, we have an error because of the float result of the operation count.index / length(var.folder_admin_roles) in terraform-0.12, element number cannot be a non-integer